CalculatorDemo.cls

```diff
@@ -0,0 +1,28 @@
+public class CalculatorDemo{

```

Comment:
Code is case sensitive will not work if I passed parameter as `SUM`. Also no proper error handling.
For example, what will happened if i called as `cd.calculator('div' , 5 ,2)` and `cd.calculator('sum' , 5 ,null)`

- [ ] Support case insensitivity for `operation` 
- [ ] Do proper error handling.